hedgehog/src/Hedgehog/Internal/Property.hs

```diff
@@ -306,7 +306,11 @@ newtype TestLimit =
 --
 newtype TestCount =
   TestCount Int
-  deriving (Eq, Ord, Show, Num, Enum, Real, Integral, Lift)
+  deriving (Eq, Ord, Num, Enum, Real, Integral, Lift)
+
+instance Show TestCount where
+  showsPrec p (TestCount n) =
+    showsPrec p n
```

Comment:
1. I we probably would want this for the other newtypes around `Int` as well.
1. If we are ok with dropping support for GHC `8.0.*` then we could simplify this to:
   ```haskell
     deriving Lift
     deriving newtype (Eq, Ord, Show, Num, Enum, Real, Integral)
   ```

Reply:
Good catch on `Show` 👍

How about planning for a release this month, and after doing the release drop support for GHC `8.0.*`?

Reply:
Regarding GHC 8.0.*, anything works for me. If we are not ready to drop support now, then I guess we want that code as this. So from my perspective, this is ready to merge.

Regarding `Show` instances of other newtype wraps, we might still want to add the manual instances or wait for after the release. Again, both works for me, but if now, I would do it as a separate PR.

Reply:
Perhaps, the best would be to merge this as-is, add `Show` instances of other newtypes as separate PRs if needed, and drop GHC 8.0.* after this release. If you agree, I think we can merge this as-is.

Reply:
Yes, sounds great 👍